Jenkinsfile

```diff
@@ -17,10 +17,27 @@ pipeline {
 
     stages {
         stage('Build Docker Image') {
-            steps {
-                deleteDir()
-                checkout scm
-                sh 'make build'
+            parallel {
+                stage('Windows') {
+                    agent {
+                        label "windock&&windows"
+                    }
+                    steps {
+                        deleteDir()
+                        checkout scm
+                        bat "powershell -File ./make.ps1"
+                    }
+                }
+                stage('Linux') {
+                    agent {
+                        label "docker&&linux"
```

Comment:
So we would need to remove the top-level `agent` and set the option to skip default checkout, right?

Reply:
With declarative, isn't the top level `agent` required?

Reply:
Unless you `skipDefaultCheckout`. [Reference](https://jenkins.io/doc/book/pipeline/syntax/#options)

Reply:
Use `agent none` at top level. [Reference](https://jenkins.io/doc/book/pipeline/syntax/#agent)